### PR TITLE
Fix tests and code samples checker

### PR DESCRIPTION
**Pull request by srknzl**
fixes test failures due to jet being disabled.

### CODE REVIEW

File: scripts/test-runner.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const DEV_CLUSTER_CONFIG = `
         xsi:schemaLocation="http://www.hazelcast.com/schema/config
         http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd">
         <cluster-name>dev</cluster-name>
+        <jet enabled="true"></jet>
```
Comment (srknzl) [inline scripts/test-runner.js:23]: some code samples use sql, so we need to enable it in the cluster that code samples are run against

File: test/integration/backward_compatible/sql/DataTypeTest.js [hunk 2]
```diff
@@ -59,6 +61,7 @@ describe('Data type test', function () {
     let mapName;
     const clientVersionNewerThanFive = TestUtil.isClientVersionAtLeast('5.0');
     const serverVersionNewerThanFive = TestUtil.isServerVersionAtLeast(client, '5.0');
+    const JET_ENABLED_CONFIG = fs.readFileSync(path.join(__dirname, 'jet_enabled.xml'), 'utf8');
```
Comment (srknzl) [inline test/integration/backward_compatible/sql/DataTypeTest.js:64]: all the tests that require sql need jet as enabled

File: test/integration/backward_compatible/sql/ExecuteTest.js [hunk 3]
```diff
@@ -341,8 +346,8 @@ describe('SqlExecuteTest', function () {
             }
             rows.should.have.lengthOf(entryCount);
 
-            resultSpy.callCount.should.be.eq(4);
-            serviceSpy.callCount.should.be.eq(4);
+            resultSpy.callCount.should.be.greaterThanOrEqual(4);
+            serviceSpy.callCount.should.be.greaterThanOrEqual(4);
```
Comment (srknzl) [inline test/integration/backward_compatible/sql/ExecuteTest.js:350]: after a change fetch may not get rows as full in non-last pages. see the issue for details: https://github.com/hazelcast/hazelcast/issues/19313

File: test/integration/backward_compatible/sql/ExecuteTest.js [hunk 4]
```diff
@@ -373,8 +378,8 @@ describe('SqlExecuteTest', function () {
...
```
Comment (srknzl) [inline test/integration/backward_compatible/sql/ExecuteTest.js:382]: same as above comment

**Review by mdumandag — APPROVED** [review]

**Pull request merged**